Library/Formula/bzr-fastimport.rb

```diff
@@ -24,32 +25,19 @@ def install
       system "python", *Language::Python.setup_install_args(libexec/"vendor")
     end
 
-    libexec.install Dir["*"]
-
-    # The plugin must be symlinked in bazaar/plugins to work
-    target = var/"bazaar/plugins/fastimport"
-    # we need to remove the target before symlinking it because if we don't and
-    # the symlink exists from a previous installation the new symlink will be
-    # created inside libexec instead of overriding the previous one because ln
-    # itself follows symlinks.
-    rm_rf target if target.exist?
-    ln_s libexec, target
+    (share/"bazaar/plugins/fastimport").install Dir["*"]
   end
 
   def caveats; <<-EOS.undent
     In order to use this plugin you must set your PYTHONPATH in your ~/.bashrc:
 
-      export PYTHONPATH="#{libexec}/vendor/lib/python2.7/site-packages:$PYTHONPATH"
+      export PYTHONPATH="#{opt_prefix}/libexec/vendor/lib/python2.7/site-packages:$PYTHONPATH"
```

Comment:
I think this can be done by below code during the install:
```ruby
(lib/"python2.7/site-packages").mkpath
(lib/"python2.7/site-packages/homebrew-bzr-fastimport.pth").write "#{libexec}/vendor/lib/python2.7/site-packages\n"
```